src/containers/ProfileDetails/components/Operator/index.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Select, Form, Button, Table, Input } from 'antd';
+import { DeleteOutlined } from '@ant-design/icons';
+import Modal from 'components/Modal';
+import styles from '../index.module.scss';
+
+const { Item } = Form;
+const { Option } = Select;
+
+const OperatorForm = ({ details, form }) => {
+  const [data, setData] = useState({ ...details });
+
+  const [modalVisible, setModalVisible] = useState(false);
+  const [nameForm] = Form.useForm();
+
+  useEffect(() => {
+    form.setFieldsValue({
+      serverOnlyAuthenticatedL2EncryptionNetwork: details.serverOnlyAuthenticatedL2EncryptionNetwork
+        ? 'true'
+        : 'false',
+      operatorFriendlyName: data.operatorFriendlyName || [],
+    });
+  }, [form, details, data]);
+
+  useEffect(() => {
+    setData({ ...details });
+  }, [details]);
+
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'dupleName',
+      width: 500,
+    },
+    {
+      title: 'Locale',
+      dataIndex: 'locale',
+    },
+    {
+      title: '',
+      width: 80,
+      render: item => (
+        <Button
+          title="removeName"
+          icon={<DeleteOutlined />}
+          className={styles.iconButton}
+          onClick={() => {
+            setData({
+              ...data,
+              operatorFriendlyName: data.operatorFriendlyName.filter(i => i !== item),
+            });
+          }}
+        />
+      ),
+    },
+  ];
+
+  const layout = {
+    labelCol: { span: 8 },
+    wrapperCol: { span: 15 },
+  };
+
+  return (
+    <div className={styles.ProfilePage}>
+      <Card title="Security">
+        <Item
+          label="OSEN:"
+          name="serverOnlyAuthenticatedL2EncryptionNetwork"
+          rules={[
+            {
+              required: true,
+              message: 'OSEN field cannot be empty',
+            },
+          ]}
+        >
+          <Select>
+            <Option value="true">enabled</Option>
+            <Option value="false">disabled</Option>
+          </Select>
+        </Item>
+      </Card>
+
+      <Card
+        title="Operator Name"
+        extra={
+          <Button type="solid" onClick={() => setModalVisible(true)}>
+            Add Name
+          </Button>
+        }
+      >
+        <Table
+          dataSource={data?.operatorFriendlyName}
+          columns={columns}
+          pagination={false}
+          rowKey={data?.operatorFriendlyName}
+        />
```

Comment:
you've used the exact same table and columns multiple times, make it a component and reuse it